shared/rakefiles/test.rake

```diff
@@ -1,21 +1,13 @@
 task :test do
-  sh "#{@exekube_cmd} rake xk['down live/#{@env}/locust',skip_secret_mgmt] || true"
+  locust_status = 0
+  sh "#{@exekube_cmd} rake xk['up live/#{@env}/locust',skip_secret_mgmt]" do |ok, res|
+    locust_status = res.exitstatus
+  end
+  # We want to clean up after Locust even if test failed
+  sh "#{@exekube_cmd} rake xk['down live/#{@env}/locust',skip_secret_mgmt]"
   Rake::Task[:destroy_tfstate].invoke('locust')
-
-  sh "#{@exekube_cmd} sh -c ' \
-      RETRIES=10; \
-      RETRY_COUNT=1; \
-      while [ \"$(kubectl get pods -n locust -o json 2> /dev/null | jq -r .items[] | grep -c .)\" != \"0\" ]; do \
```

Comment:
Why is this check no longer needed?

Reply:
Because `locust-istio` module is now a part of `live/#{@env}/locust`. It contains `kubernetes_namespace` that can't be destroyed unless every K8s resource in the namespace is gone. So `rake xk['down live/#{@env}/locust',skip_secret_mgmt]` does not leave any locust resources behind anymore.